cranelift/codegen/src/isa/aarch64/inst/mod.rs

```diff
@@ -808,6 +825,23 @@ pub enum Inst {
         size: VectorSize,
     },
 
+    /// Table vector lookup - single register table.
```

Comment:
Could we expand these doc comments (on `VecTbl` and `VecTbl2`) a bit? I went to the manual to understand what tbl/tbx do; but it would be helpful to summarize. Especially the `is_extension` bit that alters the way that `rd` is treated (full overwrite vs. conditional one), and the requirement that `rn2` is immediately after `rn` for `VecTbl2`.